Mathlib/NumberTheory/NumberField/Embeddings.lean

```diff
@@ -1187,12 +1193,31 @@ is a subset of `ℝ`. -/
 @[mk_iff] class IsTotallyReal (K : Type*) [Field K] [NumberField K] where
   isReal : ∀ v : InfinitePlace K, v.IsReal
 
-variable {K : Type*} [Field K] [NumberField K]
+variable (F : Type*) [Field F] [NumberField F] {K : Type*} [Field K] [NumberField K]
```

Comment:
Why don't you declare `F` implicit straight away? It seems to me that this is what you want in `IsTotallyReal.ofRingEquiv`  and `F` does not appear in `nrComplexPlaces_eq_zero_iff`.